package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "create-stylelint",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Oh, this change should be unnecessary since `np` (`npm run release`) automatically will update the `version` field.

```suggestion
  "version": "0.2.0",
```